### PR TITLE
DS-685 - radio card group

### PR DESCRIPTION
**Pull request by monikaracka**

### CODE REVIEW

File: packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    const view = new RadioCardGroupObject().render();
+
+    view.assertCheckedRadio();
+  });
+
+  it("should check radio after card click", async () => {
+    const view = new RadioCardGroupObject().render();
+
+    await view.clickCard(0);
+    expect(view.radios[0]).toBeChecked();
```
Comment (zuzannanap) [inline packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx:56]: why don't you use here assertCheckedRadio?

File: packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  });
+
+  it("should check only one radio card after click", async () => {
+    const view = new RadioCardGroupObject().render();
+
+    await view.clickCard(0);
+    expect(view.radios[0]).toBeChecked();
```
Comment (zuzannanap) [inline packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx:63]: ad here too?

**Review by zuzannanap — APPROVED** [review]

File: packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx (hunk 1 above)
Comment (monikaracka) [reply packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx:56]: In this test case, I want to check only one radio after click.

File: packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx (hunk 2 above)
Comment (monikaracka) [reply packages/design-system/src/components/RadioCardGroup/__tests__/RadioCardGroup.spec.tsx]: Here I changed it 😄

**Pull request merged**